addons/account/models/account_journal_dashboard.py

```diff
@@ -664,6 +714,9 @@ def action_create_vendor_bill(self):
             'views': [[view_id, 'form']],
         }
 
+    def action_open_payment_items(self):
+        return self.env['ir.actions.actions']._for_xml_id('account.action_open_payment_items')
```

Comment:
You still need to filter on the right journal, otherwise you see the lines of all journals.

I think this can be done easily by specifying `action_name` in the context, and using `open_action` instead (see `_select_action_to_open`)